server/schedule/placement/fit.go

```diff
@@ -66,11 +67,8 @@ func (f *RegionFit) Replace(srcStoreID uint64, dstStore *core.StoreInfo, region
 		return true
 	}
 
-	peers := newFitPeer(f.regionStores, region, fit.Peers, replaceFitPeerOpt(srcStoreID, dstStore))
```

Comment:
why remove peers?

Reply:
The `newFitPeer` will allocate memory to calculate the isolation score , it is too waste. So I use the `isolationStoreScore` avoid to allocate memory.